cookbooks/omnibus-supermarket/recipes/ssl.rb

```diff
@@ -71,8 +72,19 @@
     end
   end
 
+  if ! File.exists?(ssl_dhparam)
```

Comment:
Since you're doing `:create_if_missing`, you don't need to wrap this in an if statement.